vendors/watteco/models/th_v4/model.yaml

```diff
@@ -0,0 +1,36 @@
+# Commercial name of the model
+name: TH
```

Comment:
Could be named TH v4 to distinct the TH models. Now TH v3 and TH v4 would be supported.